quickfixj-examples/executor/src/main/java/quickfix/examples/executor/Application.java

```diff
@@ -350,7 +350,7 @@ public void onMessage(quickfix.fix44.NewOrderSingle order, SessionID sessionID)
         Price price = getPrice(order);
 
         quickfix.fix44.ExecutionReport accept = new quickfix.fix44.ExecutionReport(
-                    genOrderID(), genExecID(), new ExecType(ExecType.NEW), new OrdStatus(
+                    genOrderID(), genExecID(), new ExecType(ExecType.TRADE), new OrdStatus(
```

Comment:
Shouldn't this be `NEW` still? Cumulated qty is 0.

Reply:
yes